.github/workflows/build.yml

```diff
@@ -0,0 +1,141 @@
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Cache Godel assets
+      uses: actions/cache@v2
+      with:
+        path: ~/.godel
+        key: ${{ runner.os }}-godel-${{ hashFiles('godelw', 'godel/config/godel.yml') }}
+        restore-keys: |
+          ${{ runner.os }}-godel-
+
+    - name: Verify
+      run: ./godelw verify --apply=false
+
+  Dist:
+    runs-on: ubuntu-latest
+    needs:
+      - Verify
+
+    env:
+      GOPATH: /home/runner/go
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: Cache Godel assets
+      uses: actions/cache@v2
+      with:
+        path: ~/.godel
+        key: ${{ runner.os }}-godel-${{ hashFiles('godelw', 'godel/config/godel.yml') }}
+        restore-keys: |
+          ${{ runner.os }}-godel-
+
+    - name: Set up Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12
+
+    - name: Cache node_modules
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node-modules-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-node-modules-
+
+    - name: Install frontend dependencies
+      run: yarn install
+
+    - name: Build frontend
+      run: yarn run build:production
+
+    - name: Build distribution
+      run: ./godelw dist
+
+    - name: Build Docker image
+      run: ./godelw docker build --verbose
+
+    - name: Archive distribution
+      uses: actions/upload-artifact@v2
+      with:
+        name: dist
+        path: |
+          build/policy-bot/*/bin/*.tgz
+
+    #
+    # Steps after this point should only run when publishing
+    # Include them here to avoid exporting the Docker container as an artifact
+    #
+
+    - name: Login to Docker Hub
+      if: ${{ github.event_name == 'push' }}
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Push snapshot image to Docker Hub
+      if: ${{ github.ref == 'refs/heads/develop' }}
+      run: ./godelw docker push --tags=snapshot
+
+    - name: Push release image to Docker Hub
+      if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      run: ./godelw docker push --tags=latest,version
+
+    - name: Find release
+      if: ${{ startsWith(github.ref, 'refs/tags/') }}
+      id: find-release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        curl -sS --fail -H "Authorization: Bearer ${GITHUB_TOKEN}" "${GITHUB_API_URL}/repos/${GITHUB_REPOSITORY}/releases/tags/${GITHUB_REF#refs/tags/}" \
+          | jq .upload_url \
+          | sed 's/{.*}$//' \
+          | xargs -0 printf "::set-output name=upload_url::%s\n"
```

Comment:
This feature of github actions is pretty cool! [Docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idoutputs) if anyone else wants to read more.